sensor_fusion/src/sensor_fusion.h

```diff
@@ -118,7 +125,12 @@ class SensorFusionApplication
 
     bool use_interface_mgr_;
 
-    std::string inertial_frame_name_, body_frame_name_, ned_frame_name_;
+    std::string inertial_frame_name_, body_frame_name_, ned_frame_name_,
+      earth_frame_name_, global_pos_sensor_frame_name_, local_pos_sensor_frame_name_;
+
+    cav_msgs::ExternalObject bsm_obj_;
```

Comment:
If this is just a singular `ExternalObject`  instance, does the BSM processing step work for multiple objects simultaneously still?

Reply:
That's just a left over object for debugging bsm fusion. I'll take it out. All the BSMs in the queue will be processed still.